cmd/gardener-extension-runtime-gvisor/app/app.go

```diff
@@ -19,6 +19,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/gardener/gardener-extension-runtime-gvisor/pkg/version"
+
```

Comment:
Remove the empty line so that this import to be part of the block with modules import from `github.com/gardener/gardener-extension-runtime-gvisor`.

```suggestion
```

Reply:
Done. Thanks for spotting it!